server/database/queries/deletePost.js

```diff
@@ -0,0 +1,10 @@
+const connection = require('../config/connection');
+
+const deleteEvent = postId => connection.query('DELETE from event where event.id = $1 RETURNING *', [postId]);
```

Comment:
either uppercase or lower case with the SQL but not both (be consistent), and it's better if you write it with uppercase